desktop/core/src/desktop/js/apps/editor/components/aceEditor/acePredict.ts

```diff
@@ -64,7 +64,9 @@ export const attachPredictTypeahead = (
 
   const updatePredictTypeahead = () => {
     if (activePredictPromise) {
-      activePredictPromise.cancel();
+      try {
+        activePredictPromise.cancel();
+      } catch {}
```

Comment:
Catches to silence potential hic-cups?

Reply:
Indeed, I realized we added the ENABLE_PREDICT flag to silence some errors when not configured properly. We can drop this once it becomes more stable.